invisible_cities/detsim/sensor_utils.py

```diff
@@ -11,6 +11,27 @@
 from .. io      .mcinfo_io import get_sensor_binning
 
 
+def create_timestamp(event_number: Union[int, float],
```

Comment:
I think you'll need to include `Union` from `typing`, I'm getting a fail on the tests at least.

Reply:
Yes, I need to
`from typing import Union`

Now the pytest does'nt break.